hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/ReadableAirdropStore.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.token;
+
+import com.hedera.hapi.node.base.PendingAirdropId;
+import com.hedera.hapi.node.base.PendingAirdropValue;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Provides read-only methods for interacting with the underlying data storage mechanisms for
+ * pending Airdrop states.
+ */
+public interface ReadableAirdropStore {
+    /**
+     * Fetches an {@link PendingAirdropValue} object from state for given {@link PendingAirdropId}. If the airdrop contains only NFT return {@code null}.
+     * If the airdrop  could not be fetched because the given airdrop doesn't exist, returns {@code null}.
+     *
+     * @param airdropId given airdrop id
+     * @return {@link PendingAirdropValue} object if successfully fetched or {@code null} if the airdrop doesn't exist
+     */
+    PendingAirdropValue getFungibleAirdropAmount(@NonNull final PendingAirdropId airdropId);
```

Comment:
Need to change this to `get` to follow the same pattern as other stores

Reply:
Renamed